src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/utils.ts

```diff
@@ -123,3 +123,14 @@ export const areDefaultFiltersSelected = (
   if (!transportModes || transportModes.length === 0) return false;
   return transportModes.every((tm) => tm.selectedAsDefault === tm.selected);
 };
+
+export const isVenue = (location: Location): boolean => {
+  return location.resultType === 'search' && location.layer === 'venue';
```

Comment:
Result type might not be related to a venue, perhaps only `location.layer === 'venue'` must be in the utility function.

Reply:
Okey, will look at it👍